shlr/tree-sitter/lib/src/stack.c

```diff
@@ -171,7 +171,8 @@ static bool stack__subtree_is_equivalent(Subtree left, Subtree right) {
 static void stack_node_add_link(StackNode *self, StackLink link, SubtreePool *subtree_pool) {
   if (link.node == self) return;
 
-  for (int i = 0; i < self->link_count; i++) {
+  int i;
```

Comment:
didn't you say that your PR was not touching tree-sitter?

Reply:
nope, i didnt said this

Reply:
this is not needed. remove it please. tree-sitter is already compiled with `-std=c99` so it does not complain. https://github.com/radareorg/radare2/blob/master/shlr/Makefile#L334